tktitler.py

```diff
@@ -181,61 +308,18 @@ def get_period(prefix, postfix, gfyear):
     2010
     >>> get_period("", "1011", 2018)
     2010
+
     If both prefix and postfix are given, the prefix is subtracted from
     the postfix, and the gfyear is ignored:
     >>> get_period("O", "2016", 2030)
     2013
     """
 
-    prefix = prefix.upper()
-    if not re.match(r'^([KGBOT][0-9]*)*$', prefix):
-        raise ValueError("Invalid prefix: %r" % prefix)
-    if not re.match(r'^([0-9]{2}){0,2}$', postfix):
-        raise ValueError("Invalid postfix: %r" % postfix)
+    root, period = parse('%sFORM%s' % (prefix, postfix), gfyear)
+    return period
 
-    if not postfix:
-        period = gfyear
-    else:
-        if len(postfix) == 4:
-            first, second = int(postfix[0:2]), int(postfix[2:4])
-            # Note that postfix 1920, 2021 and 2122 are technically ambiguous,
-            # but luckily there was no BEST in 1920 and this script hopefully
-            # won't live until the year 2122, so they are not actually
-            # ambiguous.
-            if postfix == '2021':
-                # TODO: Should '2021' be parsed as 2020/21 or 2021/22?
-                raise NotImplementedError(postfix)
-            if (first + 1) % 100 == second:
-                # There should be exactly one year between the two numbers
-                if first > 56:
-                    period = 1900 + first
-                else:
-                    period = 2000 + first
-            elif first in (19, 20):
-                # 19xx or 20xx
-                period = int(postfix)
-            else:
-                raise ValueError(postfix)
-        elif len(postfix) == 2:
-            year = int(postfix[0:2])
-            if year > 56:  # 19??
-                period = 1900 + year
-            else:  # 20??
-                period = 2000 + year
-        else:
-            raise ValueError(postfix)
 
-    # Now evaluate the prefix:
-    prefix_value = dict(K=-1, G=1, B=2, O=3, T=1)
-    grad = 0
-    for base, exponent in re.findall(r"([KGBOT])([0-9]*)", prefix):
-        exponent = int(exponent or 1)
-        grad += prefix_value[base] * exponent
-
-    return period - grad
-
-
-def parse_bestfu_alias(alias, gfyear):
+def parse_bestfu_alias(alias, gfyear=None):
```

Comment:
Jo, lad os fjerne get_period og parse_bestfu_alias. Måske skal tktitler have en funktion der kan fortælle om noget er en BEST-titel eller FU-titel, men det er ikke sikkert. (Vi kan også godt forsvare at simplificere mailsystemet og fjerne dets behov for at klassificere om en titel er BEST eller FU.)